### PR TITLE
Councillors & committees

### PR DESCRIPTION
**Pull request by mbattad**
Please suggest more specific duties for your respective committees if you have them!

If you don't want to write out a formal list, you can just comment the general gist and I'll whip something up for you

### CODE REVIEW

**Review by lylearcinas — DISMISSED** [review]
Looks good to me, there isn't anything else I would add to the new director roles. Internal looks good, but now I'm wondering if we should add some more points to it as now the duties look quite sparse. Something I can think about tomorrow

**Comment by mbattad** [comment]
> Looks good to me, there isn't anything else I would add to the new director roles. Internal looks good, but now I'm wondering if we should add some more points to it as now the duties look quite sparse. Something I can think about tomorrow

yeah for sure! i can't reply to your comment specifically for some reason but i'm not too opposed to some roles just having smaller time commitments than others in general (e.g., Promotions is similar to what i did before in a social media coordinator role that was kinda advertised as an entry-level position for that same reason). all depends on what kind of skillset the role demands and how that fits into the CSSA exec as a whole

**Comment by Marshmallows7920** [comment]
Do we want to have a definition made for "officer" as executive officer is used in some places. Can either be changed to director or executive.

File: Constitution.md [hunk 1]
```diff
@@ -122,21 +121,18 @@ If there is more then 1 president, they exist as a single voting member on counc
...
-12. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
+### Director of Internal Affairs
 
+1. Ensure the Council is kept updated on ongoing council initiatives.
+2. Reserve the meeting room for CSSA executive meetings, general meetings and the CS Executive Assembly.
+3. Prepare an agenda and council package for executive meetings and distribute them to the Council at least one week prior to the meeting.
+4. Record meeting minutes during CSSA executive meetings and CS Executive Assemblies, or appoint another Council member to do so.
+5. Work with the President(s) to assign any actionable tasks that come about in meetings to members of Council.
+6. Work with the Director of Events to reserve rooms, equipment, and/or catering for on-campus activites.
+7. Monitor applications for Councillor positions and present them for consideration at CSSA executive meetings.
```
Comment (noahc3) [inline Constitution.md]: Is this internal's job? should it just be done by the leads of the relevant committee?

File: Constitution.md [hunk 2]
```diff
@@ -145,43 +141,65 @@ If there is more then 1 president, they exist as a single voting member on counc
...
 2. Act as a liaison between students and Council.
-3. Stay up to date on departmental and Senate meetings pertaining to the Computer Science program.
-4. Lead relevant campaigns on issues affecting the Computer Science Student Community.
-5. Work with the Director of Advocacy to promote opportunities of initiative/programs/organizations pertaining to where the Computer Science community is underrepresented.
-6. Ensure that the CSSA lounge is a safe and inclusive space.
-7. Lead the Outreach Committee with up to 3 committee members.
-8. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
+3. Stay up to date on Faculty and Senate meetings pertaining to the Computer Science program.
+4. Attend Student Executive Assembly (SEA) Meetings and Department of Computer Science Meetings on behalf of the CSSA, or put fair effort into acquiring a proxy from Council.
+5. Summarize the relevant contents of external meetings, including but not limited to those listed above, to Council.
```
Comment (noahc3) [inline Constitution.md]: nit: can we just mention the meetings by name or refer directly to points 3 and 4.

File: Constitution.md [hunk 3]
```diff
@@ -145,43 +141,65 @@ If there is more then 1 president, they exist as a single voting member on counc
...
+6. Lead relevant campaigns on issues affecting the Computer Science Student Community.
```
Comment (noahc3) [inline Constitution.md]: nit:

```suggestion
6. Lead relevant campaigns on issues affecting computer science students.
```

unless Computer Science Student Community is an actually defined group somewhere.

File: Constitution.md [hunk 4]
```diff
@@ -145,43 +141,65 @@ If there is more then 1 president, they exist as a single voting member on counc
...
+7. Work with the Director of Internal Affairs to maintain communication with other University of Manitoba student groups.
+8. Work with the Director of Advocacy to promote opportunities of initiative/programs/organizations pertaining to where the Computer Science community is underrepresented.
+9. Ensure that the CSSA lounge is a safe and inclusive space.
+10. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
 
 ### Director of Events
 
 1. Oversee the planning the events for the Computer Science community.
```
Comment (noahc3) [inline Constitution.md]:
```suggestion
1. Oversee the planning of events for the computer science community.
```

File: Constitution.md [hunk 5]
```diff
@@ -145,43 +141,65 @@ If there is more then 1 president, they exist as a single voting member on counc
...
 2. Coordinate a minimum of one event or academic initiative in each term, as per the discretion of the Council.
```
Comment (noahc3) [inline Constitution.md]:
```suggestion
2. Coordinate a minimum of one event or academic initiative in each academic term, at the discretion of the Council.
```

(sorry I know these tweaks are unrelated to the amendment and I should have caught them earlier)

File: Constitution.md [hunk 6]
```diff
@@ -145,43 +141,65 @@ If there is more then 1 president, they exist as a single voting member on counc
...
 3. Ensure there are an adequate number of social and academic events.
-4. Oversee and create posters and advertisements for all CSSA events.
-5. Ensure posters and advertisements for CSSA events are in an accessible format.
-6. Work in collaboration with other groups to promote and assist with collabroative events and initiatives.
-7. Ensure that the CSSA lounge is a safe and inclusive space.
-8. Lead the Events and Promotions Committee with up to 3 committee members.
+4. Work in collaboration with other groups to promote and assist with collaborative events and initiatives.
+5. Ensure that the CSSA lounge is a safe and inclusive space.
+6. Work with the Director of Promotions to create posters and advertisements for all CSSA events.
+7. Ensure posters and advertisements for CSSA events are in an accessible format.
```
Comment (noahc3) [inline Constitution.md]:
```suggestion
7. Ensure advertisements in an accessible format are created for all CSSA events.
```

it is not possible to make all forms of advertising accessible, as long as there are _some_ forms of accessible advertising I think that is sufficient.

File: Constitution.md [hunk 7]
```diff
@@ -145,43 +141,65 @@ If there is more then 1 president, they exist as a single voting member on counc
...
-2. Attend Community Assemblies.
-3. Guides the council on advocacy for underrepresented communities.
-4. Elected by the student body in the CSSA elections.
-5. If not filled by an elected executive in the CSSA elections, An elected executive of an UMSU recognized underrepresented community within the Department of Computer Science may be appointed as the Director of Advocacy for the CSSA. Eg: The Women in Computer Science Council.
-6. Ensure that the CSSA lounge is a safe and inclusive space.
-7. Lead the Community Advocates commitee with up to and including committee members for each community recogized under UMSU.
+2. Attend UMSU Community Assemblies.
+3. Guide the council on advocacy for underrepresented communities.
+4. Ensure that the CSSA lounge is a safe and inclusive space.
+5. Oversee the election of Community Advocates and ensure that it is run competently, fairly, and transparently to CSSA's members.
```
Comment (noahc3) [inline Constitution.md]: nit:

```suggestion
5. Oversee the election of Community Advocates and ensure that it is run competently, fairly, without bias, and transparently to CSSA's members.
```

(maybe idk :shrug:)

File: Constitution.md [hunk 8]
```diff
@@ -145,43 +141,65 @@ If there is more then 1 president, they exist as a single voting member on counc
...
+6. Work with, call meetings with, and delegate tasks to Community Advocates.
+7. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
+
+### Director of Finances
+1. Act as a signing authority for CSSA.
+2. Providing a brief report on CSSA finances and expenses at each meeting.
```
Comment (noahc3) [inline Constitution.md]: nit:

```suggestion
2. Provide a brief report on CSSA finances and expenses at each meeting.
```

but also are we sure we want to mandate this in the constitution? every single meeting?

File: Constitution.md [hunk 9]
```diff
@@ -145,43 +141,65 @@ If there is more then 1 president, they exist as a single voting member on counc
...
+3. Be responsible for all funds generated by Council events and for overseeing all funds spent by Council.
```
Comment (noahc3) [inline Constitution.md]: i think I would split this into multiple points

- Be responsible for and track all funds generated by Council events.
- Oversee and track all expenditures generated by Council.
- Approve or deny purchase requests generated by Council with discretion and with the best interests of the CSSA in mind.

Im not sure I really like how I worded the 2nd and 3rd points there but hopefully someone gets the gist and can rephrase it better lol

File: Constitution.md [hunk 10]
```diff
@@ -145,43 +141,65 @@ If there is more then 1 president, they exist as a single voting member on counc
...
+4. Make available all audits of CSSA to any Student members upon request with more than two weeks of notice.
```
Comment (noahc3) [inline Constitution.md]: just a question but what audits? just the year end audits we do ourselves? we cant afford real 3rd party audits. not sure if there are audits by umsu or something.

File: Constitution.md [hunk 11]
```diff
@@ -145,43 +141,65 @@ If there is more then 1 president, they exist as a single voting member on counc
...
+5. Perform a year-end audit at the end of the CSSA Fiscal Year.
+6. Prepare and present the budgets for group activities.
+7. Contact UMSU, the Science Students' Association, and external sponsors for funding when appropriate.
```
Comment (noahc3) [inline Constitution.md]: I think also add this point to events director (but just "for events funding")

File: Constitution.md [hunk 12]
```diff
@@ -207,6 +225,42 @@ If there are no written minutes, written notice, and/or the information presente
...
 If an executive officer is dismissed, the club must follow [**Section 3: Club Changeover**](#section-3-club-changeover) in order to follow the proper procedure(s) outlined to fill the vacant position(s).
 
+## Article 2d: Committees, Councillors, and Community Advocates
+
+### Committees
+A Committee is a group of council members that forms to complete a well-defined goal. Any Executive Officer may propose a committee by outlining its purpose and the criteria of its members (if any). The Council will vote on the formation of this Committee, and upon its formation, the proposing Executive Officer will lead it.
+
+A Committee leader will have the following duties:
+1. Appoint Committee members from Council.
+2. If applicable, announce the formation of the Committee to Members of the Association so that they may apply for it.
```
Comment (noahc3) [inline Constitution.md]: nit: should "Members of the Association" be capitalized? maybe just "members of the CSSA"?

File: Constitution.md [hunk 13]
```diff
@@ -207,6 +225,42 @@ If there are no written minutes, written notice, and/or the information presente
 
 If an executive officer is dismissed, the club must follow [**Section 3: Club Changeover**](#section-3-club-changeover) in order to follow the proper procedure(s) outlined to fill the vacant position(s).
 
+## Article 2d: Committees, Councillors, and Community Advocates
+
+### Committees
+A Committee is a group of council members that forms to complete a well-defined goal. Any Executive Officer may propose a committee by outlining its purpose and the criteria of its members (if any). The Council will vote on the formation of this Committee, and upon its formation, the proposing Executive Officer will lead it.
+
+A Committee leader will have the following duties:
```
Comment (noahc3) [inline Constitution.md]: nit:
```suggestion
A Committee leader has the following duties:
```

File: Constitution.md [hunk 14]
```diff
@@ -207,6 +225,42 @@ If there are no written minutes, written notice, and/or the information presente
...
+6. Report on Committee activites and progress at Council meetings.
+7. Appoint a proxy Committee leader in the event that they become unavailable to lead.
+
+If a Committee leader deems the Committee's goal to be fulfilled, they may propose its dissolution to be voted on by the Council.
+
+### Councillors
+Councillors are non-voting members of council. Members of CSSA may apply for a Councillor position through an application that is publicly available throughout the year. Councillor applicants may indicate their interest to join an active committee. Once appointed by the Council, a Councillor may assume a general purpose volunteer role, or be appointed to a committee that they express interest in.
+
+### Community Advocates
+Applicants will have the option to self-nominate for a Community Advocate position on their Councillor application. However, self-nominations for Community Advocates will close [WITHIN SOME TIME LIMIT], after which the option will be removed from application form.
```
Comment (noahc3) [inline Constitution.md]: question: is this a separate form from councillors?

ps. don't forget to fill in the time limit :P

File: Constitution.md [hunk 15]
```diff
@@ -207,6 +225,42 @@ If there are no written minutes, written notice, and/or the information presente
...
+- 2SLGBTQIA+ Students' Advocate
+- Indigenous Students' Advocate
+- Black Students' Advocate
+- International Students' Advocate
+- Mature Students' Advocate
+- Racialized Students' Advocate
+
+An applicant must self-identify as a member of the community they are self-nominating to advocate for.
+
+If multiple applicants self-nominate for the same Community Advocate role, the role will be voted on by CSSA members that self-identify as members of that community. The Director of Advocacy is responsible for overseeing and administering this process.
```
Comment (noahc3) [inline Constitution.md]: what if we dont have members who self-identify in that community?

File: Constitution.md (hunk 8 above)
Comment (Marshmallows7920) [reply Constitution.md]: I think yes, even if the update is "no update this week" we should. A week is a bit time from finance perspective so accountability for them to staying up to date on bank account or any frauds etc.

File: Constitution.md (hunk 1 above)
Comment (Marshmallows7920) [reply Constitution.md]: Internal is making sure of it. They can choose to delegate to others if they want

File: Constitution.md (hunk 2 above)
Comment (Marshmallows7920) [reply Constitution.md]: I assume to cover changes and additional ones that come up, eg the new science one

File: Constitution.md (hunk 3 above)
Comment (Marshmallows7920) [reply Constitution.md]: Maybe a more accurate would be parts of the CS community that are affected by it? Or is this like a united we stand so if it affects some we all go to war

File: Constitution.md (hunk 7 above)
Comment (Marshmallows7920) [reply Constitution.md]: Does "fairly" umbrella it? Should we add "accessibly"

File: Constitution.md (hunk 14 above)
Comment (Marshmallows7920) [reply Constitution.md]: Can we arbitrarily define that these should be done before the start of community assemblies or is that too vague

File: Constitution.md (hunk 10 above)
Comment (Marshmallows7920) [reply Constitution.md]: Yes umsu has audits, and I think we would be transparent enough that it shouldn't come up as much from students but still can have there

File: Constitution.md (hunk 15 above)
Comment (Marshmallows7920) [reply Constitution.md]: It's ok to have a vacant role. Even the umsu mature rep role has been vacant for a few years. Giving the option from someone of the community to represent would be ideal but if not the other advocates would likely still be working together to make sure everyone is included

File: Constitution.md (hunk 2 above)
Comment (mbattad) [reply Constitution.md]: Yeah my thought was that there may be some that aren't covered in points 3 and 4, I could make it less wordy by saying "external meetings" and cutting it off there

File: Constitution.md (hunk 3 above)
Comment (mbattad) [reply Constitution.md]: Agree that "Computer Science Student Community" sounds like an official thing the way it's layed out here. I kinda prefer "computer science students"

File: Constitution.md (hunk 14 above)
Comment (mbattad) [reply Constitution.md]: 1. I was thinking the same form but with a question for self-nom, then that question gets removed after the time limit.
2. "before community assemblies" is a touch vague for my liking, how about tentatively end of september? I'll put that in for now and we'll see what thoughts people have

File: Constitution.md (hunk 15 above)
Comment (mbattad) [reply Constitution.md]: Agree w/Isham, I'll put that in more explicitly ty!

Edit: just realized you probably meant if there are no community members to elect that position- in that case I think it's ok to leave it up to the Director of Advocacy to appoint a position? that imo falls under "overseeing the process" but we can make it clearer if that's a concern. a little bit worried about tying future execs' hands

File: Constitution.md (hunk 1 above)
Comment (mbattad) [reply Constitution.md]: Councillor apps can be independent of committees in my mind's eye so used Internal to cover that case. Ultimately if a committee lead decides they want to do the monitoring/presenting/etc for their own committee, that's no problem! Internal should be the one taking the reponsibility if nobody else volunteers though, agree

File: Constitution.md (hunk 8 above)
Comment (mbattad) [reply Constitution.md]: I'll say "regular updates" as to not overregulate, how about?

File: Constitution.md (hunk 9 above)
Comment (mbattad) [reply Constitution.md]: Last point I ended up rewording as "potential expenditures" and also saying "*members of the* CSSA" for disambiguation

File: Constitution.md (hunk 10 above)
Comment (mbattad) [reply Constitution.md]: Mostly had our personal audits in mind even though it's very "just take my word for it." Good to know that UMSU does audits as well
